lms/djangoapps/bulk_email/tasks.py

```diff
@@ -430,7 +431,11 @@ def format_address(course_title_no_quotes):
     # but with the course name rather than course title.
     # Amazon SES's from address field appears to have a maximum length of 320.
     __, encoded_from_addr = forbid_multi_line_headers('from', from_addr, 'utf-8')
-    if len(encoded_from_addr) >= 320 and truncate:
+
+    # It seems that this value is also escaped when set out to amazon, judging
+    # from our logs
+    escaped_encoded_from_addr = cgi.escape(encoded_from_addr)
```

Comment:
Cool, I'll switch to that. Note that I'm not actually escaping anything — that's happening somewhere between django and amazon. I'm just trying to reverse engineer the escaping that's making the character limit exceed 321